packages/xstate-test/src/testUtils.ts

```diff
@@ -0,0 +1,22 @@
+import { StatePlan } from '@xstate/graph';
```

Comment:
this file is only used for tests - it would make sense to move it there cause right now it looks like it's a public API when it's declared in `src`, unless... this is a public API?

Reply:
Yes, this should be moved to `/tests/testUtils.ts`

Reply:
@davidkpiano This would mean it would be run as part of the tests, but I can add an ignore to the jest config.

Reply:
We can do this in a follow-up PR

Reply:
> @davidkpiano This would mean it would be run as part of the tests, but I can add an ignore to the jest config.

Tests in Jest (by default) are only executed for **all** files in `__tests__` and for all files with the `.test.(j|t)sx?` extension~. Our tests are located in a `test` directory and all of them have `.tests.tsx?` extension - so it's as easy as moving this to a test directory. This won't be recognized as a test file because it doesn't comply with its rules regarding the containing dir and the extension pattern